website/docs/guides/core-to-cloud-1.md

```diff
@@ -0,0 +1,244 @@
+- [Git setup](https://docs.getdbt.com/guides/core-to-cloud-1?step=6#git-setup): Learn to link your dbt project's Git repository with dbt Cloud.
+- [Developer setup:](https://docs.getdbt.com/guides/core-to-cloud-1?step=7#developer-setup) Understand the setup needed for developing in dbt Cloud.
+- [Orchestration setup](https://docs.getdbt.com/guides/core-to-cloud-1?step=8#orchestration-setup): Learn how to prepare your dbt Cloud environment and jobs for orchestration.
+- [Models configuration](https://docs.getdbt.com/guides/core-to-cloud-1?step=9#models-configuration): Get insights on validating and running your models in dbt Cloud, using either the dbt Cloud IDE or dbt Cloud CLI.
+- [What's next?](https://docs.getdbt.com/guides/core-to-cloud-1?step=10#whats-next): Summarizes key takeaways and introduces what to expect in the following guides.
+
+## Prerequisites
+
+- You have an existing dbt Core project connected to a data platform and Git repository.
+- You have a dbt Cloud account. **[Don't have one? Start your free trial today](https://www.getdbt.com/signup)**!
+
+### Related docs
+- [Learn dbt Cloud](https://courses.getdbt.com/collections)
+- [Develop with dbt Cloud](/docs/cloud/about-develop-dbt)
+- [Deploy jobs](/docs/deploy/deployments)
+- Book [expert-led demos](https://www.getdbt.com/resources/dbt-cloud-demos-with-experts) and insights
+- Work with the [dbt Labs’ Professional Services](https://www.getdbt.com/dbt-labs/services) team to support your data organization and migration.
+
+## Account setup
+
+1. [Create your dbt Cloud account](https://www.getdbt.com/signup).
+
+2. Learn about [access](/docs/cloud/manage-access/about-user-access) and [invite users](/docs/cloud/manage-access/about-user-access) to your dbt Cloud account and dbt Cloud project.
+
+3. Configure [Single Sign-On (SSO)](/docs/cloud/manage-access/sso-overview) or [Role-based access control (RBAC)](/docs/cloud/manage-access/about-user-access#role-based-access-control) for easy and secure access. <Lifecycle status='enterprise' />
+   - This removes the need to save passwords and secret environment variables locally.
+
+4. In **Account settings**, switch on [partial parsing](/docs/deploy/deploy-environments#partial-parsing) to only reparse changed files, saving time.
+
+5. In **Account settings**, enable [Git repo caching](/docs/deploy/deploy-environments#git-repository-caching) to ensure job reliability, protection from third-party outages, and performance. <Lifecycle status='enterprise' />
+
+## Data platform setup
+
+This section explains the considerations and methods to connect your data platform to dbt Cloud.
+
+1. Set up your [data platform connections](/docs/cloud/connect-data-platform/about-connections) and [environment variables](/guides/core-to-cloud-1?step=4) in dbt Cloud. dbt Cloud can connect with a variety of data platform providers including:
```

Comment:
Might help the experience if the links are consistent in their destinations rather than skipping them ahead in the guide. 

```suggestion
1. Set up your [data platform connections](/docs/cloud/connect-data-platform/about-connections) and [environment variables](/docs/build/environment-variables) in dbt Cloud. dbt Cloud can connect with a variety of data platform providers including:
```